### PR TITLE
close #14 add startJob with File and InputStream in SparkJobServerClientImpl

### LINKED ISSUE #14: Is there way to run job with config or data file?
Is there way to run job with config or data file?

Now, there isn't way to run job directly with a file or a input stream.
However, in actual, frequently we'd like  to run job with config or data file.

Shall we add runJob method with a file or input stream to SparkJobServiceClientImpl?

### PR DESCRIPTION
**Pull request by yatszhash**
add startJob with File and InputStream in SparkJobServerClientImpl.

### CODE REVIEW

File: src/main/java/org/khaleesi/carfield/tools/sparkjobserver/api/SparkJobServerClientImpl.java [hunk 1]
```diff
@@ -1,16 +1,12 @@
 package org.khaleesi.carfield.tools.sparkjobserver.api;
 
-import java.io.BufferedInputStream;
-import java.io.Closeable;
-import java.io.File;
-import java.io.FileInputStream;
-import java.io.FileNotFoundException;
-import java.io.IOException;
-import java.io.InputStream;
+import java.io.*;
```
Comment (bluebreezecf) [inline src/main/java/org/khaleesi/carfield/tools/sparkjobserver/api/SparkJobServerClientImpl.java]: xx.xxx.* is not a good way to import class, please use the explicit xxx.xxxx.YYY import sentence instead.
Using Ctrl+shift+o in eclipse jdt can do it for you.

**Review by bluebreezecf — COMMENTED** [review]
@yatszhash  Please check my review  comments, and make the corresponding modification.

File: pom.xml [hunk 2]
```diff
@@ -4,10 +4,9 @@
   <artifactId>spark-job-server-client</artifactId>
   <version>1.0.0</version>
   <name>spark-job-server-client</name>
-  <description>Java Lib for Spark Job Server Rest API</description>
-  
-  <dependencies>
+	<description>Java Lib for Spark Job Server Rest API</description>
```
Comment (bluebreezecf) [inline pom.xml]: The content seems doesn't change, so this file can be checkout with the old format

**Comment by yatszhash** [comment]
I'm sorry for my lack of care.
I appreciate it.

**Comment by bluebreezecf** [comment]
@yatszhash   The code seems good now. Thanks for your contribution!

**Pull request merged**